api-reference/10 UI Components/dxDrawer/1 Configuration/openedStateMode.md

```diff
@@ -18,7 +18,11 @@ The drawer overlaps the view.
 The view's width decreases to accommodate the drawer.
 
 - *"push"*      
-The drawer partially displaces the view.
+The drawer partially displaces the view. 
+
+In *"push"* and *"overlap"* modes, Drawer calculates its height based on the content height. In the *"shrink"* mode, Drawer uses its template's height for the same purpose.
```

Comment:
Why does this information matter to the developer?

Reply:
Drawer's ways to calculate its size are sometimes unobvious, escpecially in some openened state modes. This can cause issues like the following one: https://supportcenter.devexpress.com/internal/ticket/details/T862540#